certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -402,8 +413,8 @@
         "consumes": [
           "application/json"
         ],
-        "description": "Convert any type of office documents to PDF format. https://pdf4me.com/power-automate/documentation/actions/convert-to-pdf/",
-        "operationId": "ConvertToPdf_V1",
```

Comment:
@varunynoox The operation has been deleted. This change breaks existing flows. 
Can you please confirm that no users will be affected by this change?

Reply:
operation id has been restored with commit - https://github.com/microsoft/PowerPlatformConnectors/pull/2335/commits/685ab73dd86143b823f5c9b44bda06904cbe0508